actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsBackgroundActor.java

```diff
@@ -93,7 +93,11 @@ private void orgCreationMetricsData(Request actorMessage) {
       csvRecords.add(headers);
       for (String operation : OrganisationMetricsUtil.operationList) {
         String requestStr = getRequestObject(operation, requestId);
-        String ekStepResponse = makePostRequest(JsonKey.EKSTEP_CONTENT_SEARCH_URL, requestStr);
+        String baseSearchUrl = System.getenv(JsonKey.SEARCH_SERVICE_API_BASE_URL);
+        if (StringUtils.isBlank(baseSearchUrl)) {
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.